### ISSUE TITLE
'Compiling for the Web' section is outdated and confusing users

### ISSUE THREAD

**Issue by capnm** [godotengine/godot-docs#3900]
**Your Godot version:**
3.2.x
**Issue description:**
The build structure or instructions of current emscripten changed,
e.g the mentioned file **.emscripten** isn't involved any more.  
https://emscripten.org/docs/getting_started/downloads.html

**URL to the documentation page:**
https://docs.godotengine.org/en/latest/development/compiling/compiling_for_web.html

**Comment by jeronimo-schreyer** [comment]
I also tried latest emscripten version (2.0.9) on Linux and is not working with latest godot 3.2.4 version. so where it states that one should use
``
emsdk activate latest
``
you actually need to use
``
emsdk activate 1.40.1
``
which is the latest 1.x version

**Comment by Faless** [comment]
> I also tried latest emscripten version (2.0.9) on Linux and is not working with latest godot 3.2.4 version

Yeah, they broke compatibility in their latest version, the 2.0.8 should also work.

**Comment by jeronimo-schreyer** [comment]
tried version 2.0.8 and didn't work either. Checked the version using emsdk list, I'm also using precompiled node-12.18.1-64bit (don't know if that changes anything)

2.0.7 linked correctly though

I'm using target=release tools=no flags to build Godot

**Comment by Calinou** [comment]
Closing in favor of https://github.com/godotengine/godot-docs/issues/11758, which covers more recent information that is now outdated.
